scorecard/profile_data.py

```diff
@@ -639,6 +639,21 @@ def get_indicators(api_data):
     return indicators
 
 
+def abs_median(items):
+    return median([abs(x) for x in items])
```

Comment:
Hmm. I think what we should be doing here is finding the index of the median element (absolute values), then return that actual element (without the median).

A bit like a `decorate / sort / undecorate` pattern, but applied to calculating the median. My reasoning is that the negative value informs whether it's under or overspending, it doesn't actually impact the *degree* of over/underspending (which is what we're trying to get when we calculate the median).